src/structures/Message.js

```diff
@@ -223,6 +223,17 @@ class Message extends Base {
           messageID: data.message_reference.message_id,
         }
       : null;
+
+    /**
+     * The message this message replies to
+     * @type {?Message}
+     */
```

Comment:
this needs to be directly above `this.replyReference = ...` or have an `@name Message#replyReference` tag